### PR TITLE
[multistep] Activations in client 

### PR DESCRIPTION
**Pull request by ataymano**

### CODE REVIEW

File: include/live_model.h [hunk 1]
```diff
@@ -275,6 +275,17 @@ namespace reinforcement_learning {
...
+    /**
+    * @brief Report that action was taken.
+    *
+    * @param primary_id  The unique primary_id used when choosing an action should be presented here.  This is so that
+    *                  the action taken can be matched with feedback received.
+    * @param secondary_id Index of the partial outcome.
+    * @param status  Optional field with detailed string description if there is an error
+    * @return int Return error code.  This will also be returned in the api_status object
+    */
+    int report_action_taken(const char* primary_id, const char *secondary_id, api_status* status = nullptr);
```
Comment (zwd-ms) [inline include/live_model.h:287]: whitespace format in `const char*`

File: rlclientlib/live_model.cc [hunk 2]
```diff
@@ -239,6 +239,17 @@ namespace reinforcement_learning
...
-    return _pimpl->request_episodic_decision(event_id, previous_id, context_json, resp, episode, status);
+    return _pimpl->request_episodic_decision(event_id, previous_id, context_json, action_flags::DEFAULT, resp, episode, status);
   }
+
+  int live_model::request_episodic_decision(const char* event_id, const char* previous_id, const char* context_json, unsigned int flags, ranking_response& resp, episode_state& episode, api_status* status) {
+    INIT_CHECK();
+    return _pimpl->request_episodic_decision(event_id, previous_id, context_json, flags, resp, episode, status);
+  } 
+
+  int live_model::report_action_taken(const char* primary_id, const char *secondary_id, api_status* status) {
```
Comment (zwd-ms) [inline rlclientlib/live_model.cc:250]: whitespace format in `const char*`

**Review by zwd-ms — APPROVED** [review]

File: include/live_model.h (hunk 1 above)
Comment (ataymano) [reply include/live_model.h]: fixed

File: rlclientlib/live_model.cc (hunk 2 above)
Comment (ataymano) [reply rlclientlib/live_model.cc]: fixed

**Pull request merged**